app/src/main/java/com/algo/phantoms/algo_phantoms/activities/SplashScreenActivity.kt

```diff
@@ -1,12 +1,31 @@
 package com.algo.phantoms.algo_phantoms.activities
 
+import android.content.Intent
+import android.os.Build
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.os.Handler
+import android.view.WindowInsets
+import android.view.WindowManager
 import com.algo.phantoms.algo_phantoms.R
 
 class SplashScreenActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_splash_screen)
+        @Suppress("DEPRECATION")
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            window.insetsController?.hide(WindowInsets.Type.statusBars())
+        } else {
+            window.setFlags(
+                    WindowManager.LayoutParams.FLAG_FULLSCREEN,
+                    WindowManager.LayoutParams.FLAG_FULLSCREEN
+            )
+        }
+        @Suppress("DEPRECATION")
```

Comment:
do not use deprecated methods migrate this code to `Handler(Looper.getMainLooper()).postDelayed({},2500)`